handlers/message.js

```diff
@@ -20,59 +21,16 @@ const sendForApproval = async (text, channel_id, user_id, hash) => {
     channel: "at-channel-requests",
     text: "There's a new at-channel request!",
     blocks: [
-      {
-        type: "section",
-        text: {
-          type: "mrkdwn",
-          text: `:wave: Hello, kind moderators!\n\n<@${user_id}> has requested to use at-channel in <#${channel_id}>. The message is:`
-        }
-      },
-      {
-        type: "section",
-        text: {
-          type: "mrkdwn",
-          text: `>>>${text}`
-        }
-      },
-      {
-        type: "divider"
-      },
-      {
-        type: "section",
-        text: {
-          type: "mrkdwn",
-          text: "Do you want to *approve* or *reject* this message?"
-        }
-      },
+      getMarkdownSection(`:wave: Hello, kind moderators!\n\n<@${user_id}> has requested to use at-channel in <#${channel_id}>. The message is:`),
+      getMarkdownSection(`>>>${text}`),
+      { type: "divider" },
+      getMarkdownSection("Do you want to *approve* or *reject* this message?"),
```

Comment:
Thoughts on building this function to allow an array of data passed in?

Reply:
Sure, that sounds like a great idea for a future, further enhancement.